contracts/ReleaseValidator.sol

```diff
@@ -28,28 +43,31 @@ contract ReleaseValidator {
   )
     public
     view
-    returns (bool)
+    returns (uint8 code)
   {
-    require(address(packageDb) != 0x0, "escape:ReleaseValidator:package-db-not-set");
-    require(address(releaseDb) != 0x0, "escape:ReleaseValidator:release-db-not-set");
-
-    if (!validateAuthorization(packageDb, callerAddress, name)) {
+    if (address(packageDb) == 0x0){
+      // packageDb address is null
+      return 1;
```

Comment:
These error numbers seem like they should be an [`Enum`](https://solidity.readthedocs.io/en/v0.4.24/structure-of-a-contract.html?highlight=enum#enum-types).  That would make reading this code a lot easier.

Reply:
Ah yes nice! 🙂